### ISSUE TITLE
Wordcount error (report = "warn")

### ISSUE THREAD

**Issue by Kalif-Vaughn** [crsh/rmdfiltr#5]
Hello,

I'm trying to knit a document in R and I'm receiving this error:

```
Error in rmdfiltr::add_wordcount_filter(args, report = "warn") : 
  unused argument (report = "warn")
Calls: <Anonymous> -> <Anonymous> -> pdf_pre_processor
Execution halted
```
I'm using papaja but as far as I know I have not changed anything to cause this error.

Thanks for any assistance.

**Comment by crsh** [comment]
Hi there, yes, sorry this is a breaking change introduced in a recent version. Installing the latest development version of `papaja`  should fix the problem (`devtools::install_github("crsh/papaja@devel")`). Let me know if you continue to experience this problem.

**Comment by Kalif-Vaughn** [comment]
Thanks for your response! Reinstalling worked. :)

I'll recount a bit of my experience in case it is useful for others. I kept running into an error with "fansi". If it wasn't installed, it would say papaja couldn't be installed. If I installed fansi, it would say papaja couldn't be installed because it 'could not remove a prior version of fansi'. The solution was to download RTools first, then install fansi, then install papaja and select "1" to update all packages.
